### PR TITLE
Eip 55 verification

### PR DESCRIPTION
**Pull request by KoushikGavini**
Checksum Verification Function based on EIP 55

### CODE REVIEW

File: chain-connect/package.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
   "version": "1.2.4",
   "dependencies": {
     "@gala-chain/api": "1.2.4",
-    "ethers": "^6.12.1",
```
Comment (befitsandpiper) [inline chain-connect/package.json:6]: revert this change

File: chain-api/package.json [hunk 2]
```diff
@@ -9,7 +9,6 @@
     "class-transformer": "0.5.1",
     "class-validator": "^0.14.0",
     "class-validator-jsonschema": "^5.0.0",
-    "elliptic": "^6.5.4",
```
Comment (befitsandpiper) [inline chain-api/package.json:12]: why did you remove this dependency?

File: chain-test/package.json [hunk 3]
```diff
@@ -6,15 +6,10 @@
   "dependencies": {
     "@gala-chain/client": "1.2.4",
     "nanoid": "^3.3.6",
-    "tslib": "^2.6.2",
-    "@jest/globals": "29.7.0"
+    "tslib": "^2.6.2"
```
Comment (befitsandpiper) [inline chain-test/package.json:9]: removal of dependencies are causing errors

**Comment by dzikowski** [comment]
I strongly recommend to keep the previous implementation of `checksumedEthAddress`. Any change there requires a pretty solid justification.

**Comment by dzikowski** [comment]
will be covered also in #321
